### PR TITLE
Hotfix/0.4.2

### PR DESCRIPTION
**Pull request by pavelmaksimov25**
https://spryker.atlassian.net/browse/SDK-976

### CODE REVIEW

**Review by sergeyspryker — APPROVED** [review]

**Review by vol4onok — APPROVED** [review]

File: src/Extension/ValueResolver/ReportDirValueResolver.php [hunk 1]
```diff
@@ -42,6 +42,10 @@ public function getValue(ContextInterface $context, array $settingValues, bool $
 
         $reportDir = $settingValues[static::REPORT_DIR_SETTING];
 
+        if (!is_dir($reportDir)) {
+            mkdir($reportDir, 0777, true);
```
Comment (dereuromark) [inline src/Extension/ValueResolver/ReportDirValueResolver.php:46]: we never 0777, we always 0770 etc

**Comment by dereuromark** [comment]
Please use normal branch stategy here for demos
We release normally unless there is a fix to an older version (major)

File: src/Extension/ValueResolver/ReportDirValueResolver.php (hunk 1 above)
Comment (pavelmaksimov25) [reply src/Extension/ValueResolver/ReportDirValueResolver.php:46]: Ok. Where I can find this rule?
I also don't like `0777`, but I couldn't find any RFC and it is already implemented almost everywhere in sdk. It is kinda default permission to deal with shared files and directories.

Comment (dereuromark) [reply src/Extension/ValueResolver/ReportDirValueResolver.php:46]: I see
Fair enough then

**Pull request closed without merging**